frontend/messages/fr.json

```diff
@@ -68,7 +68,7 @@
   "summaryTotalCloudCover": "Couverture nuageuse",
   "summaryRainfall": "Précipitations",
   "summaryCumuliDepth": "Épaisseur des cumulus",
-  "summarySoaringLayerDepth": "Couche convective",
+  "summarySoaringLayerDepth": "Epaisseur couche convective",
```

Comment:
This message appears in a place where space is expensive:

![image](https://github.com/soaringmeteo/soaringmeteo/assets/332812/b5ba890c-a0c3-4b65-b012-42770ab72c94)

I would recommend keeping “Couche convective” or maybe abbreviating to “Épaisseur couche conv.”

Reply:
```suggestion
  "summarySoaringLayerDepth": "Épaisseur couche conv.",
```

Preview:

![image](https://github.com/soaringmeteo/soaringmeteo/assets/332812/476066cc-f814-46a6-9ae8-118cb6d5634e)